### PR TITLE
Switch release to maven central

### PR DESCRIPTION
**Pull request by dougwettlaufer**

### CODE REVIEW

**Review by dimas-b — APPROVED** [review]

**Review by mpenick — DISMISSED** [review]

**Review by dimas-b — DISMISSED** [review]

**Review by dimas-b — DISMISSED** [review]

File: persistence-cassandra-4.0/pom.xml [hunk 1]
```diff
@@ -1,243 +1,218 @@
...
-            <version>4.3.0</version>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>io.reactivex.rxjava2</groupId>
-            <artifactId>rxjava</artifactId>
-            <version>2.2.15</version>
-        </dependency>
-        <dependency>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (dimas-b) [inline persistence-cassandra-4.0/pom.xml:2]: Would it be too much trouble to move reformatting into a separate PR?

Comment (dougwettlaufer) [reply persistence-cassandra-4.0/pom.xml:2]: Not possible. Parent pom pulls in the format plugin which means I have to make it happy or else I can't merge the PR.

Comment (olim7t) [reply persistence-cassandra-4.0/pom.xml:2]: Not necessarily for this, but just FYI you can disable the plugins with Maven flags: `-Dfmt.skip=true -Dxml-format.skip=true`.

**Review by olim7t — APPROVED** [review]
